inventory/sample.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -11,31 +11,75 @@ hosting_infrastructure: aws
 # Cluster Environment ID to uniquely identify the environment
 env_id: "<REPLACE WITH VALID ENV ID - i.e: env1>"
 
-## AWS Provisioning Variables
-# Amazon image name to use for instances.
-# Note that each AWS region may utilize a different AMI for the same image.
-aws_image_name: <REPLACE WITH VALID AMI>
-
-# AWS instance flavors (i.e.: instance sizes)
-master_flavor: m4.xlarge
-infra_node_flavor: i3.xlarge
-app_node_flavor: m4.xlarge
-cns_node_flavor: i3.xlarge
-
-# Number of master instances to deploy.
-# This is a non-HA sample nventory so don't change this value
-aws_num_masters: 1
-
-# Number of instances to be used as infra nodes.
-# This is a non-HA sample inventory so don't change this value
-aws_num_infra_nodes: 1
-
-# Number of instances to be used as compute/app nodes
-aws_num_app_nodes: 3
-
-# Number of instances to be used as CNS (Container Native Storage) nodes
-aws_num_cns_nodes: 0
-
+# Define custom Master(s) API adn Console Port - Default 8443
+#openshift_master_api_port: 443
```

Comment:
Leaving these commented out causes errors. Looks like the default values are not being populated:

```
Setting Common Facts ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.02s


Failure summary:


  1. Hosts:    localhost
     Play:     localhost
     Task:     Ensure the underlaying infrastructure is {{ action }}
     Message:  The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'etcdnodes'
               
               The error appears to have been in '/tmp/src/casl-ansible/roles/manage-aws-infra/tasks/manage-cf-stack.yml': line 2, column 3, but may
               be elsewhere in the file depending on the exact syntax problem.
               
               The offending line appears to be:
               
               ---
               - name: Ensure the underlaying infrastructure is {{ action }}
                 ^ here
               We could be wrong, but this one looks like it might be an issue with
               missing quotes.  Always quote template expression brackets when they
               start a value. For instance:
               
                   with_items:
                     - {{ foo }}
               
               Should be written as:
               
                   with_items:
                     - "{{ foo }}"
```

Reply:
Done, will push new commit soon

Reply:
@makentenza Awesome. Ping me and I will test it out.